src/frontend/src/store/index.ts

```diff
@@ -86,11 +89,21 @@ export default new Vuex.Store({
         })
         .then((response) => {
           if(response.data.success) {
-            alert("Transaction was successful!");
-            dispatch("ACTION_FETCH_WALLET_AMOUNT");
-          } else if(response.data.err) {
-            alert("Transaction has failed.");
-          }
+            Vue.$toast.success('Transaction has sent!', {
+              message: 'Transaction has sent!',
+              duration: 3000,
+              position: 'top',
+              dismissible: true,
+            });
+            commit("MUTATION_SET_WALLET_AMOUNT", response.data.remaining_balance);
+          } 
+        }, () => {
+          Vue.$toast.error('Transaction has failed.', { 
```

Comment:
Maybe add some sort of info for the user as to why the transaction failed? Insufficient funds, wrong wallet ID, etc. Maybe we can just have the toast display the error message that we get from the response?